data/lua/carryover_gold.lua

```diff
@@ -47,83 +52,108 @@ local function get_popup_text_basic()
 	return title, report
 end
 
-local function do_carryover_gold()
+--- Calculates the amount of carryover gold.
```

Comment:
Please make these documentation comments conform to the format of the documentation comments in other modules, for example `location_set` or `functional`. That means using `@param` and `@return` annotations in addition to 3 dashes for the brief description as you've done here.